src/utils/Fields/renderPlaceholder.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
@drewdrewthis I'm hesitant to delete because it's really a utility function and doesn't belong within the file for the select field itself